bin/httk-conf

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+DIRNAME="$(dirname "$0")"
+HTTK_BIN_PATH="$(cd "$DIRNAME"; pwd -P)"
+
+echo $HTTK_BIN_PATH
```

Comment:
Please remind me why this is needed? What path are you trying to uncover, because as we move things around for httk to be properly installed by pip and anaconda, things that you may try to access via this directory may not be where you expect them to be.

That said, it isn't a bad idea with a script that prints out httks "configuration", much like some other software has, e.g.: `gpgconf`. But we probably want to adopt a more expandable format for that, e.g., such as `gpgconf` printing out a key name and a value:
```
gpg:OpenPGP:/usr/bin/gpg
gpg-agent:Private Keys:/usr/bin/gpg-agent
scdaemon:Smartcards:/usr/lib/gnupg/scdaemon
gpgsm:S/MIME:/usr/bin/gpgsm
dirmngr:Network:/usr/bin/dirmngr
pinentry:Passphrase Entry:/usr/bin/pinentry
``` 

But, a reasonable key for the path you want to access depends on what you intended to use it for...

Reply:
I've looked in the ADAQ code; you need this specifically to get the path to the httk helper binaries. Alright, lets just name this binpath. I can fix this on both the httk and ADAQ sides if you want.

Reply:
```suggestion
#!/bin/bash

DIRNAME="$(dirname "$0")"
HTTK_BIN_PATH="$(cd "$DIRNAME"; pwd -P)"

echo "binpath:$HTTK_BIN_PATH"
```